docs/extensibility/form-section.md

```diff
@@ -398,6 +399,21 @@ GenericList widgets render an array as a list of collapsible sections with their
   children:
     - path: '[].host'
     - path: '[].port'
+
+ - widget: GenericList
+   path: spec.filter.filters
+   children:
+     - path: '[].eventType.value'
+       placeholder: placeholder.eventType
+     - path: '[].eventSource.value'
+   template:
+     eventSource:
+     property: source
+       type: exact
+       value: ''
+     eventType:
+       property: type
+       type: exact
```

Comment:
Could you add one tab before `property`, please? I believe it should be at the same level as `type` and `value`.